block/provider/util.go

```diff
@@ -293,3 +294,13 @@ func SetRetryParameters(maxAttempts int, maxGap int) {
 func roundUpSize(volumeSizeBytes int64, allocationUnitBytes int64) int64 {
 	return (volumeSizeBytes + allocationUnitBytes - 1) / allocationUnitBytes
 }
+
+// validateIKSSession check if IKS session is valid
+func validateIKSSession(vpcs *VPCSession) (err error) {
+	if vpcs.Config == nil {
```

Comment:
Putt code comments //IKS Session is invalid contains nil configuration.Please check if IAM token exchange request for IKS failed.